Replace `malibu` with `csrf-csrf` (#1)
awidener3/roosevelt: README.md

[Cross-Site Request Forgery](https://en.wikipedia.org/wiki/Cross-site_request_forgery) protection is enabled by default in Roosevelt and will be triggered on all routes that handle POST requests. The CSRF protection works by defining a trusted token when a page with a form is loaded and then verifying that that token is the same one that was given when the form is submitted.

> **Roosevelt automatically protects any request other than GET, HEAD, or OPTION.**

Reviewer: ```suggestion
Roosevelt automatically protects any request other than GET, HEAD, or OPTION.
```

Here's an example of defining a CSRF token on a page that serves up a form:

```js
router.route('/page-with-a-form').get(app.get('csrfProtection'), app.get('csrfErrorHandling'), (req, res) => {
  const model = { }
  res.render('page-with-a-form', model)
})
```

Then you will need to include the `csrfProtection` middleware in your POST as well:

```js
router.route('/post-route').post(app.get('csrfProtection'), app.get('csrfErrorHandling'), (req, res) => {
  res.send('success')
})
```

> That's it. When you submit the form, Roosevelt checks the CSRF token and confirms a match, then you won't see a 403 Forbidden error.

Reviewer: ```suggestion
That's it. When you submit the form, Roosevelt checks the CSRF token and confirms a match. If it doesn't match, a 403 Forbidden error will be displayed.
```
